theano/tensor/elemwise.py

```diff
@@ -890,14 +891,15 @@ def perform(self, node, inputs, output_storage):
             # numpy the first (faster) version leads to segfaults
             if self.ufunc:
                 ufunc = self.ufunc
+            elif not hasattr(node.tag, 'ufunc'):
+                # It happen that make_thunk isn't called, like in
+                # get_scalar_constant_value
+                self.prepare_node(node, None, None, 'py')
+                if self.ufunc:
+                    ufunc = self.ufunc
```

Comment:
self.prepare_node() can add to self.ufunc or node.tag.ufunc depending if it can be reused or not. So we need to check it again. I'll add a comment to make this clear.